packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -338,8 +349,16 @@ class IrisGridUtils {
       invertSearchColumns = true,
       pendingDataMap = [],
       frozenColumns,
+      columnHeaderGroups,
     } = irisGridState;
     const { columns, formatter } = model;
+
+    let hydratedColumnHeaderGroups: ColumnHeaderGroup[] | null = null;
+
+    if (columnHeaderGroups) {
```

Comment:
Nope. If the user has deleted all `columnHeaderGroups`, it will be set to an empty array. The setter method handles empty arrays fine (as well as undefined for clearing). `null` (or undefined) is used for the "no overrides" value

Reply:
Ah okay well then just `if (columnHeaderGroups != null) {`

I was more confused why the `strict-boolean-expressions` isn't tripping here, I thought it would require `!= null`